src/py/flwr/driver/driver.py

```diff
@@ -35,6 +44,146 @@
 """
 
 
+def start_driver(  # pylint: disable=too-many-arguments
```

Comment:
We could move this into a new module called `app` (to mimic the structure we have in `flwr.server` and `flwr.client`.